modules/ROOT/pages/deploy-to-cloudhub.adoc

```diff
@@ -79,7 +79,11 @@ The undeploy command also deletes the app in Mule Maven plugin 3.3.0 and later v
 
 include::mule-runtime::partial$mmp-concept.adoc[tag=authenticationMethodsIntro]
 
+<<<<<<< HEAD
 include::mule-runtime::partial$mmp-concept.adoc[tags=!authenticationMethodsIntro;!authenticationRTF;!authenticationOnPrem]
+=======
+include::mule-runtime::partial$mmp-concept.adoc[tags=authenticationOptionsList;!authenticationRTF;!authenticationOnPrem]
+>>>>>>> v4.3
```

Comment:
TODO: which is correct to keep in v4.4?

* from v4.4: contains `tags=!authenticationMethodsIntro`
* from v4.3: contains `tags=authenticationOptionsList`

Reply:
@Cristian-Venticinque

Reply:
Same as before, line 85 is correct.